tests/test_ext_napoleon_docstring.py

```diff
@@ -1302,6 +1302,32 @@ def test_parameters_with_class_reference(self):
         expected = """\
 :param param1:
 :type param1: :class:`MyClass <name.space.MyClass>` instance
+"""
+        self.assertEqual(expected, actual)
+
+    def test_multiple_parameters(self):
+        docstring = """\
+Parameters
+----------
+x1, x2 : array_like
+    Input arrays, description of ``x1``, ``x2``.
+
+"""
+
+        config = Config(napoleon_use_param=False)
+        actual = str(NumpyDocstring(docstring, config))
+        expected = """\
+:Parameters: **x1, x2** (*array_like*) -- Input arrays, description of ``x1``, ``x2``.
```

Comment:
this might have to change after merging #8049 (if you merge this first I'll update that PR).

Reply:
I rebased this into 3.x branch now. And it seems going well without a fix.

Reply:
exactly, but #8049 removes the restriction on the type preprocessor to only run when `napoleon_use_param = True`. So once you merge this PR I'm going to have to update #8049 to use this (with `napoleon_type_aliases = {}`) instead:
```python
:Parameters: **x1, x2** (:class:`array_like`) -- Input arrays, description of ``x1``, ``x2``.
```

Reply:
Oh, I'd forgotten to merge #8049. hehe.

Reply:
updated.